bytecheck/src/lib.rs

```diff
@@ -283,6 +284,16 @@ impl<C: ?Sized> CheckBytes<C> for PhantomPinned {
     }
 }
 
+impl<C: ?Sized, T: CheckBytes<C>> CheckBytes<C> for ManuallyDrop<T> {
```

Comment:
Nit: This can be `T: CheckBytes<C> + ?Sized`

Reply:
ya